### ISSUE TITLE
Crashes on AndroidX

### ISSUE THREAD

**Issue by searchy2** [ittianyu/BottomNavigationViewEx#101]
Library does not work with AndroidX support libraries. Please update to use AndroidX, thanks!

**Comment by searchy2** [comment]
Crashes with error:
```
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
        at com.ittianyu.bottomnavigationviewex.BottomNavigationViewEx.getBottomNavigationItemViews(BottomNavigationViewEx.java:572)
        at com.ittianyu.bottomnavigationviewex.BottomNavigationViewEx.enableAnimation(BottomNavigationViewEx.java:344)
        at com.ittianyu.bottomnavigationviewexsample.features.style.StyleActivity.init(StyleActivity.java:29)
        at com.ittianyu.bottomnavigationviewexsample.features.style.StyleActivity.onCreate(StyleActivity.java:25)
        at android.app.Activity.performCreate(Activity.java:6975)
```

**Comment by thanhduy26091995** [comment]
The same issue, please update it

**Comment by searchy2** [comment]
Fixed in #103

**Comment by dpgc11** [comment]
The version 2.0.2 still is not updated to Android X and it's crashing
